internal/nginx/main.go

```diff
@@ -88,15 +89,39 @@ func NewPostStatusRequest(path, contentType string, data interface{}) (int, []by
 	return res.StatusCode, body, nil
 }
 
+// GetServerBlock takes an nginx.conf file and a host and tries to find the server block for that host
+func GetServerBlock(conf string, host string) (string, error) {
```

Comment:
Did you leave this here because it'll be needed later on when we add more commands to `dbg`?

Reply:
Maybe. I'm also making use of it in the actual plugin, and I feel that it's good to keep it next to the other common nginx.conf reading stuff.